src/pages/Swap/Swap.tsx

```diff
@@ -813,6 +812,29 @@ const AlternativesContent: FC<{ clickNo: number; setClickNo: (n: number) => void
   const [tokens, setTokens] = useState([])
   const [details, setDetails] = useState([])
 
+  const swapAndFindBestPrice = (details) => {
+    if (details[1] && details[1]?.name.toLowerCase().includes('goosefx')) {
+      details[1] = {
+        ...details[1],
+        fastest: true
+      }
+      const swap = details[0]
+      details[0] = details[1]
+      details[1] = swap
+    }
+
+    let best = 0
+    for (let i = 0; i < details.length; i++) {
```

Comment:
You do not need to loop through the entire array as it is already sorted by best price, so it should just be comparison between the first and second if goosefx is present or picking the first one if not present